requirements/requirements.txt

```diff
@@ -2,11 +2,12 @@ accelerate
 asyncio
 deepspeed>=0.14.0
 deepspeed-kernels
+fastapi
 Flask-RESTful
 grpcio
 grpcio-tools
 Pillow
-pydantic
+pydantic==1.*
```

Comment:
Hi @svaruag - FYI we are working on getting PyDantic v2 support added, some comments on @adk9's PR [here](https://github.com/microsoft/DeepSpeed-MII/pull/480) but let us know if you need to merge this first.